src/rss_etl.py

```diff
@@ -64,39 +108,49 @@ def extract_metadata_from_rss_feed(self, max_attempts=3):
             if bill is None:
                 continue
 
-            bill_no, printer_no = self.get_bill_and_printer_no(bill.title)
-
-            self.create_and_attempt_to_insert_bill(bill, bill_no)
-            self.create_and_attempt_to_insert_revision(bill, bill_no, printer_no)
+            bill_identifier = BillIdentifier(bill["guid"])
+            self.create_and_attempt_to_insert_bill(bill, bill_identifier)
+            self.create_and_attempt_to_insert_revision(bill, bill_identifier)
             attempt_count += 1
             if attempt_count >= max_attempts:
                 break
 
-    def create_and_attempt_to_insert_revision(self, bill, bill_no, printer_no):
-        revisions_output_record = self.create_revisions_output_record(bill, bill_no, printer_no)
-        if not revisions_output_record.has_existing_supabase_records(["bill_id", "printer_no", "legislative_body"]):
+    def create_and_attempt_to_insert_revision(self, bill, bill_identifier: BillIdentifier):
```

Comment:
NIT

Using "upsert" may be clearer and more concise. I.e. `def upsert_revision`. You can apply similar renames to other functions if desired

Reply:
"upsert" would not be quite descriptive of what's going on here -- there's no update occurring if a row already exists.